RogueTech Core/GameTips/general.txt

```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
 Upgrade the Argo! Don't scoff at morale upgrades or ignore your Mechtechs.
 Choose Mechwarriors who work well as a team and complement each other's skills.
 Equipment is key. A well equipped Light can kill a poorly equipped Assault.
-Instability removes Evasion AND Guarded! Perfect for a follow up volley.
+Instability removes Evasive AND Guarded! Perfect for a follow up volley.
 You can have up to three repair bays working simultaneously with Argo upgrades, and at 100% efficiency to boot!
 Ammo explodes. Don't have too much of it. Also, don't run out of ammo. But you do you, chief.
-Untrained Mechwarriors are more likely to panic. Punch through their armor to make them reach for the Eject button. 
+Untrained Mechwarriors are more likely to panic. Punch through their armor to make them reach for the eject button. 
 Scare your enemy into bailing out! Destroying their allies, flanking, and reducing their offensive capability all lead to panic.
 The danger zones in a DropShip landing area are no joke, no matter what your fellow Mechwarriors say.
 Plan your lance according to the range & strategy you intend to use.
 A 'Mech needs an engine core to function. 
 An XL engine will save a lot of weight, but if you lose a Torso Side, you lose the 'Mech.
-Bring some streaks or pulse lasers to deal with those pesky Lights!
+Weapons with accuracy bonuses or evasion ignoring, like Pulse Lasers, are fantastic against fast-moving enemies.
 Watch your heat. Riding the redline will slow you down or blow you sky high.
-Riding the redline is dangerous... but fun.
+Riding the redline is dangerous...but fun.
```

Comment:
```suggestion
Riding the redline is dangerous... but fun.
```
As far as I know, an ellipse with following text gets a space between the two.

Reply:
This depends entirely on the style guide employed for writing. I've reached out to HBS to see if their writers can comment on what style guide was used for HBS BattleTech. When I get a response, I'll adopt whatever it is that they're using.

In the meantime, I'll leave this as [UK (ONS) style](https://style.ons.gov.uk/house-style/ellipsis/) for internal consistency.